examples/gno.land/r/sys/validators/gnosdk.gno

```diff
@@ -1,24 +1,31 @@
 package validators
 
 import (
-	"sort"
-
 	"gno.land/p/sys/validators"
 )
 
 // GetChanges returns the validator changes stored on the realm, since the given block number.
-// This function is unexported and intended to be called by gno.land through the GnoSDK
+// This function is intended to be called by gno.land through the GnoSDK
 func GetChanges(from int64) []validators.Validator {
-	// Perform binary search to find the starting index
-	idx := sort.Search(len(changes), func(i int) bool {
-		return changes[i].blockNum >= from
-	})
-
-	// Construct the changes from the found index
 	valsetChanges := make([]validators.Validator, 0)
-	for i := idx; i < len(changes); i++ {
-		valsetChanges = append(valsetChanges, changes[i].validator)
-	}
+
+	// Gather the changes from the specified block
+	changes.Iterate("", "", func(_ string, value interface{}) bool {
```

Comment:
you should be able to do `changes.Iterate(strconv.Itoa(from), "", ...)`

Reply:
TIL:

[a027d94](https://github.com/gnolang/gno/pull/2229/commits/a027d940ce48200af7d33f5a61a1987c8aecdead)